.github/workflows/container-image-buildah.yml

```diff
@@ -47,18 +47,37 @@ permissions:
 
 jobs:
   buildah:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - architecture: amd64
+            runner: ubuntu-latest
+          - architecture: arm64
+            runner: ubuntu-24.04-arm
+    runs-on: ${{ matrix.runner }}
     steps:
       - name: Sanitize Platforms
         id: platforms
         run: |
-          platforms="${{ inputs.platforms == '' && 'linux/amd64' || inputs.platforms }}"
-          archs="$( sed -e 's#linux/##g' <<< $platforms )"
+          platforms="${{ inputs.platforms == '' && 'linux/amd64,linux/arm64' || inputs.platforms }}"
+          if [ "${{ matrix.architecture }}" = "arm64" ]; then
+            platforms="linux/arm64"
+          else
+            platforms="$( sed -e 's#linux/arm64,##g' -e 's#,linux/arm64##g' -e 's#linux/arm64##g' <<< $platforms)"
+            archs="$( sed -e 's#linux/##g' <<< $platforms )"
+            echo "archs=$archs" >> $GITHUB_OUTPUT
+          fi
           echo "platforms=$platforms" >> $GITHUB_OUTPUT
-          echo "archs=$archs" >> $GITHUB_OUTPUT
+
+      - name: Install Podman on ubuntu-24.04-arm
+        if: ${{ matrix.architecture == 'arm64' }}
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y podman
 
       # Allow multi-target builds
       - name: Set up QEMU
+        if: matrix.architecture != 'arm64'
```

Comment:
Why it is an inequality check instead of something like `matrix.architecture == 'amd64'`?

Will it require rework if e.g. RISC-V runner gets added?

Reply:
I thought we needed to setup qemu on non-arm64 runner, but that doesn't  make sense as we should use amd64 for building other archs when no natively runners are available for them.

Using `matrix.architecture == 'amd64'` here makes more sense here. I have updated this line.